### ISSUE TITLE
Providing a value more than MAX_INT guarantees a roll of 1

### ISSUE THREAD

**Issue by meadsteve** [meadsteve/DiceApi#2]
oops

**Comment by jnewc** [comment]
Can we get an ETA on this? My enterprise Java app has a dependency on this free, not-for-profit API, so I demand that that this issue be prioritised immediately.

**Comment by meadsteve** [comment]
Unfortunately I spent all my initial seed money on capturing 3D models of dice being rolled in zero gravity. Which got lost during re-entry. 

As a patch I suggest rolling thousands of D3s and combing the results, adding one, then subtracting the number of dice rolled.

**Comment by jnewc** [comment]
```
int diceRoll = getDiceRollSlowSyncWay(d) + (random.nextInt() % (d + 1));
```

This fixed it for me and also feels more enterprise-y.

**Comment by meadsteve** [comment]
fixed by 80e89bbb03
